src/applications/personalization/rated-disabilities/actions/index.js

```diff
@@ -5,6 +5,9 @@ export const FETCH_RATED_DISABILITIES_SUCCESS =
 export const FETCH_RATED_DISABILITIES_FAILED =
   'FETCH_RATED_DISABILITIES_FAILED';
 
+export const FETCH_TOTAL_RATING_SUCCESS = 'FETCH_TOTAL_RATING_SUCCESS';
```

Comment:
**Suggestion:** Use past-tense verbs for action names.

From [Human Redux](https://read.reduxbook.com/markdown/part1/03-updating-state.html):

> What you choose as the name of the action sent as the `type` property can be anything. However, it's a subtle but important thing to realize that actions should be used to report a thing that happened, not cause something to happen. Again, think of them as news stories: they don't tell you how to react, they inform you of what occurred. We'll see how these reports are consumed later when we discuss reducers.

> One little "hack" to keep yourself writing actions this way is to make all your action types past-tense. So instead of calling an action `LOGIN_SUCCESS`, you'd call it `LOGIN_SUCCEEDED`. Doing that can help you avoid a few common anti-patterns down the road.

Reply:
Great feedback, addressed in https://github.com/department-of-veterans-affairs/vets-website/pull/10950/commits/605e2a3d0cfc25f9e941dbf7c81bc6337ea9ef67